distro/src/bin/hazelcast

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+PRGDIR="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)"
+export HAZELCAST_HOME=${HAZELCAST_HOME:-$(cd "$PRGDIR/.." >/dev/null; pwd -P)}
+
+#
+function find_RUN_JAVA() {
+    if [ $JAVA_HOME ] ; then
+        RUN_JAVA=$JAVA_HOME/bin/java
+    else
+        RUN_JAVA=`which java 2>/dev/null`
+    fi
+
+    if [ -z $RUN_JAVA ] ; then
+        echo "Error: Java not found. Please install Java 8 or higher in your PATH or set JAVA_HOME appropriately"
+        exit 1
+    fi
+}
+
+#############
+
+find_RUN_JAVA
```

Comment:
nit: not sure it needs to be a separate function.

Reply:
Done 👍